<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="XAV-502000046" failures="3" tests="11" name="Non-Grops method" time="61.078" errors="0" timestamp="2021-06-08T17:41:31 IST">
  <testcase classname="TestNG.parameters" name="add" time="0.006"/>
  <testcase classname="TestNG.parameters" name="dependsGroup0" time="0.0"/>
  <testcase classname="TestNG.parameters" name="dependsGroup2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNG.parameters.dependsGroup2(parameters.java:65)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- dependsGroup2 -->
  <testcase classname="TestNG.parameters" name="dependsGroup1" time="0.0">
    <skipped/>
  </testcase> <!-- dependsGroup1 -->
  <testcase classname="TestNG.parameters" name="readDataProvider" time="0.0"/>
  <testcase classname="TestNG.parameters" name="readDataProvider" time="0.0"/>
  <testcase classname="TestNG.parrallelTesting" name="display" time="20.263">
    <failure type="java.lang.AssertionError" message="expected [Gmail1] but found [Gmail]">
      <![CDATA[java.lang.AssertionError: expected [Gmail1] but found [Gmail]
at TestNG.parrallelTesting.display(parrallelTesting.java:46)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- display -->
  <testcase classname="TestNG.parrallelTesting" name="show" time="0.0">
    <skipped/>
  </testcase> <!-- show -->
  <testcase classname="TestNG.parrallelTesting" name="show1" time="0.0">
    <skipped/>
  </testcase> <!-- show1 -->
  <testcase classname="TestNG.retryFailedTestCase" name="failed" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [GMail] but found [TELUS SSO]">
      <![CDATA[java.lang.AssertionError: expected [GMail] but found [TELUS SSO]
at TestNG.retryFailedTestCase.failed(retryFailedTestCase.java:45)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- failed -->
  <testcase classname="TestNG.retryFailedTestCase" name="findCount" time="10.532"/>
</testsuite> <!-- Non-Grops method -->
